spec/scrabble_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require_relative '../lib/scrabble.rb'
```

Comment:
Thanks for this heads up, I've done that so most future code should be less painful to read.... at least in that regard. :) m